Legacy Docs Review: Error handling (#7114)
Particular/docs.particular.net: nservicebus/recoverability/configure-error-handling.md

## Error queue monitoring

> Administrators should monitor the error queue in order to detect when problems occur. The message in the error queue contains relevant information such as the endpoint that originally processed the message and exception details. This lets an administrator investigate the problem.

Reviewer: ```suggestion
Administrators should monitor the error queue in order to detect when problems occur. The message in the error queue contains relevant information such as the endpoint that initially processed the message and exception details. This allows an administrator to investigate the problem.
```

> Monitoring and handling of failed messages with [ServicePulse](/servicepulse/) provides access to full exception details including the stack-trace. [ServiceInsight](/serviceinsight/) offers advanced debugging capability providing additional information like message flow visualization. Both ServiceInsight and ServicePulse provide message retry functionality that sends a failed message back to the originating endpoint for re-processing. For more details on how to retry a message using ServicePulse, see [Introduction to Failed Messages Monitoring in ServicePulse](/servicepulse/intro-failed-messages.md). To retry a message using ServiceInsight, see [Managing Errors and Retries in ServiceInsight](/serviceinsight/managing-errors-and-retries.md).

Reviewer: ```suggestion
Monitoring and handling of failed messages with [ServicePulse](/servicepulse/) provides access to full exception details, including the stack-trace. [ServiceInsight](/serviceinsight/) provides advanced debugging capabilities by offering additional information, such as message flow visualization. Both ServiceInsight and ServicePulse provide message retry functionality, which sends a failed message back to the originating endpoint for reprocessing. For more details on how to retry a message using ServicePulse, see [Introduction to Failed Messages Monitoring in ServicePulse](/servicepulse/intro-failed-messages.md). To retry a message using ServiceInsight, see [Managing Errors and Retries in ServiceInsight](/serviceinsight/managing-errors-and-retries.md).
```

> If either ServicePulse or ServiceInsight are not available in the environment, the message retry functionality can be performed using the native management tools appropriate for the selected transport:

Reviewer: ```suggestion
If ServicePulse and ServiceInsight are not available in the environment, the message retry functionality can be performed using the native management tools appropriate for the selected transport:
```
